src/components/Header/Header.styled.tsx

```diff
@@ -0,0 +1,12 @@
+import styled from "styled-components";
+
```

Comment:
Please run `npm run lint` (and resolve warnings/errors) and `npm run format .` to fix the linter issues